src/components/Capture/Face.js

```diff
@@ -36,13 +36,25 @@ class Face extends Component {
   handleCapture = payload => {
     const { actions, nextStep } = this.props
     const id = randomId()
-    actions.createCapture({ ...defaultPayload, ...payload, id })
+    const faceCaptureData = {
+      ...defaultPayload,
+      ...payload,
+      sdkMetadata: this.addIsCrossDeviceAndDeviceTypeProperties(payload.sdkMetadata),
+      id
+    }
+    actions.createCapture(faceCaptureData)
     nextStep()
   }
 
+  addIsCrossDeviceAndDeviceTypeProperties = sdkMetadata => ({
```

Comment:
This seem to be repeated in both components...

Reply:
Good spot! I think I'll move it out into a util under `onfidoApi.js`.